xds/src/main/java/io/grpc/xds/EdsLoadBalancer.java

```diff
@@ -339,9 +324,9 @@ public boolean canHandleEmptyAddressListFromNameResolution() {
 
       @Override
       public void shutdown() {
-        loadStatsStoreMap.remove(clusterServiceName);
-        if (isReportingStats()) {
-          loadReportClient.removeLoadStatsStore(clusterServiceName);
+        if (isReportingLoad) {
+          xdsClient.cancelClientStatsReport(clusterName, null);
```

Comment:
It only works for the case cluster_service name is never changed, otherwise the first ClusterEndpointsBalancer.shutdown will cancel the second ClusterEndpointsBalancer's load report. It that intended behavior for now?